src/NuGet.Config

```diff
@@ -6,6 +6,5 @@
   <packageSources>
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="netcorenightly" value="https://www.myget.org/F/akkadotnet-netcore/api/v3/index.json" />
```

Comment:
We were restoring from the wrong package sources and the akka-netcore nightlies were hosting a different version of 0.9.5 for some reason.